charts/kube-state-metrics/templates/service.yaml

```diff
@@ -12,8 +12,8 @@ metadata:
 {{ toYaml .Values.customLabels | indent 4 }}
 {{- end }}
   annotations:
-    {{- if .Values.prometheusScrape }}
-    prometheus.io/scrape: '{{ .Values.prometheusScrape }}'
+    {{- if .Values.prometheus.monitor.enabled }}
```

Comment:
How about setting a separate variable `$promScrape` ?

It could be set as the following
```
{{- $promScrape := or .Values.prometheus.monitor.enabled .Values.prometheusScrape -}}
```

Reply:
Removing a value is honestly a breaking change, and we should be bumping major versions if we even decide to go down the route of completely replacing the parameter

Reply:
@DanielCarmel do you still plan to address this feedback? Would love to get it merged. :)

Reply:
I thought the value `prometheusScrape` is a redundant
Im removing the change around this value